### ISSUE TITLE
Enhancement: Remove (or at least improve) the potential duplicate workflow/job

### ISSUE THREAD

**Issue by AbnormalPoof** [FunkinCrew/Funkin#3943]
### Issue Checklist

- [X] I have properly named my enhancement
- [X] I have checked the Issues/Discussions pages to see if my enhancement has already been suggested

### What is your suggestion, and why should it be implemented?

The potential duplicate workflow, on paper, is a great way to check if any issues are potentially... well... duplicates.
In reality, this kinda sucks.

9 times out of 10, the bot is just blatantly incorrect:
![Screenshot 2024-12-17 at 5 59 11 PM](https://github.com/user-attachments/assets/ff75d606-06b7-4634-a122-b8974eb36152)
![Screenshot 2024-12-17 at 5 58 51 PM](https://github.com/user-attachments/assets/3fc94e1f-42dd-4749-9cd1-b3a4aa5cc5d4)
Even at high confidence (the % next to each issue), the bot is still wrong.
It doesn't help that this runs every time an issue is edited, which can lead to it commenting on 'potential duplicate' issues multiple times if one were to edit it a lot.

Basically it's not really doing much for the Issues tab right now. My suggestion is to either remove it entirely, or find some way to improve it's accuracy since it could have some potential if improved upon.

**Comment by EliteMasterEric** [comment]
Here's the action being used: https://github.com/wow-actions/potential-duplicates

I picked it up since it seemed like it would be useful but it appears it hasn't played out well in practice. Re-running on edit is definitely a configuration issue.

Since it doesn't seem like it has been helpful, I'm going to remove it in a minute. Was a good experiment at least.

**Comment by EliteMasterEric** [comment]
Done.
